### ISSUE TITLE
Support of Media Responses

### ISSUE THREAD

**Issue by hyzhak** [googlesamples/assistant-sdk-python#203]
I'm working right now on action which uses Media Responses (https://github.com/internetarchive/internet-archive-google-action) and would like to use my AIY Voice kit here. But AIY responses that it doesn't support Media Response.
Is there any plan to support Media Response in this SDK?
https://developers.google.com/actions/assistant/responses#media_responses
Could I expect it in near future?

**Comment by Fleker** [comment]
Yeah the APIs in the SDK don't support media right now. Thanks for the feedback.

**Comment by hyzhak** [comment]
You are welcomed! Could I expect it on DIY device soon or give up all hope?

**Comment by Fleker** [comment]
It's something I'd like to support, although I don't have a specific date for when it'd be ready.

**Comment by hyzhak** [comment]
Great! Thanks :)

**Comment by hyzhak** [comment]
Hi there! What's news? Is there any chance to see this feature soon?

**Comment by fdambrosio** [comment]
Hi, I have the same feature  request too,  thanks

**Comment by Fleker** [comment]
The library now supports playback of news & podcasts.

**Comment by Fleker** [comment]
You'll be able to say something like "Hey Google, play This American Life" and get the podcast.

For playing random audio files, it would be better to use a custom device action so that "Play random audio files" would get received and you would then handle that by playing a random audio file.

**Comment by hyzhak** [comment]
@Fleker sure, we are developing custom device action - https://github.com/internetarchive/internet-archive-google-action/ but recently I couldn't send Media Response, because Assistant SDK (and AIY in particular) didn't support [Media Responses](https://developers.google.com/actions/assistant/responses#media_responses). 
In other words:
```javascript
conv.surface.capabilities.has('actions.capability.MEDIA_OUTPUT') === false
``` 
Can I use them after the last update?

**Comment by Fleker** [comment]
Honestly I have not tried AoG with the library yet. If you set up the [Conversation Components](http://github.com/actions-on-google/conversation-components-nodejs) sample, you may be able to run it, ask for media, and see what happens.

**Comment by Fleker** [comment]
Did you ever see if this works?

**Comment by hyzhak** [comment]
@Fleker I have asked Google AIY team to update their software to fetch new version of Assistant SDK https://github.com/google/aiyprojects-raspbian/issues/304. so let see, will it help or not.

**Comment by dmarvp** [comment]
hi @Fleker I'm also interested in getting media responses within the assistant sdk. Are there any news related to that?
